ms2rescore/peptideshaker.py

```diff
@@ -182,10 +397,73 @@ def to_peprec(self):
         return peprec
 
     def get_search_engine_features(self):

```

Comment:
We should move this function to another module, to be shared with the maxquant pipeline, avoiding code duplication.